docs/variants/msi_z690/installation-manual.md

```diff
@@ -0,0 +1,74 @@
+# Dasharo compatible with MSI PRO Z690-A WIFI DDR4 - installation manual
+
+Flashing coreboot can be done from Linux using flashrom with the internal
+programmer. This document describes the process of building, installing and
+running flashrom on Ubuntu 21.04.
```

Comment:
Ubuntu version is inconsistent. I'm not saying to change that now but at lease we should try to align that.

Reply:
See comment above https://github.com/Dasharo/docs/pull/145#discussion_r849938870